mars_api/data_import/data_importer.py

```diff
@@ -0,0 +1,145 @@
+"""Like Transporter but without Jason Statham."""
+from sys import stdout
+
+from dateutil import parser
+from django.db.utils import IntegrityError
+
+from mars_api.serializers import (GameSerializerForImportedData,
+                                  PlayerScoreSerializer)
+
+FIELD_REMAPPING = {
+    "game": {"map": "Default"},
+    "player_score": {
+        "corporation": {
+            "Interpl. Cinematics": "Interplanetary Cinematics",
+            "UNMI": "United Nations Mars Initiative",
+        }
+    },
+}
+
+
+def str_to_bool(arg):
+    return bool(int(arg))
+
+
+GAME_FIELD_MAPPING = {
+    0: {"id": int},
+    1: {"date": parser.parse},  # datetime
+    2: {"game_map": str},
```

Comment:
`    "game": {"map": "Default"},` this part from above should already be integrated here - you will then be able to get rid of this extra logic

```
    game_dict = list_to_dict(data, GAME_FIELD_MAPPING)

    if game_dict["game_map"] == FIELD_REMAPPING["game"]["map"]:
        game_dict["game_map"] = "Tharsis"

    return game_dict
```